db/api/src/tickets.rs

```diff
@@ -1397,6 +1392,7 @@ impl HoprDbTicketOperations for HoprDb {
                                 ))
                             })?;
 
+                            // TODO: cache this DB call too, or use the channel graph
```

Comment:
The question is whether the SQLite cache is not sufficient, not only here, but in multiple tables and values that rarely change.